spring-boot-jib/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+server.port= 8086
```

Comment:
It is important to set the port to 8086 instead of the default 8080? If not, let's remove it.

Reply:
Deleted the file. Reason behind adding it is mentioned above